.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,27 +6,39 @@ labels: bug
 
 ---
 
+**⚠️IMPORTANT⚠️ if you do not fill this out, we cannot help you.**
+
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**Plaintext**
+**⚠️IMPORTANT⚠️ The below code is non-negotiable for "Ciphey didn't decrypt...." problems. If you do not tell us your plainttext, we will not help you.** 
+
+```
+Include your plaintext here, replacing this 
+```
+
+**Verbose Output**
+**⚠️IMPORTANT⚠️ Verbose outputt will tell us why it's not working the way we expected it to be.**
+Run Ciphey with `ciphey -vvv` and paste the results (either as a file, or copied & pasted) into the GitHub issue comment. 
+
+**Versions:**
+**⚠️IMPORTANT⚠️ We need this information because Ciphey does not work effectively on all systems.**
+ - OS/Distro: [e.g. Windows, Debian 11.0, Arch, OS X El Capitan]
+ - Python version: [python3 --version]
+ - Ciphey versions: [python3 -m pip show ciphey cipheycore cipheydists]
+ - Did you use Docker?
+
 **To Reproduce**
 Steps to reproduce the behavior:
 1. What input did you use?
 2. What flags / arguments did you use?
 
-**Verbose Output**
-Run Ciphey with `ciphey -vvv` and paste the results (either as a file, or copied & pasted) into the GitHub issue comment. 
-
 **Expected behavior**
 A clear and concise description of what you expected to happen.
 
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Versions:**
- - OS/Distro: [e.g. Windows, Debian 11.0, Arch, OS X El Capitan]
- - Python version: [python3 --version]
- - Ciphey versions: [python3 -m pip show ciphey cipheycore cipheydists]
-
 **Additional context**
```

Comment:
This can probably be changed to an "any other info" section, if we are rigidly enforcing a template